composer.json

```diff
@@ -3,6 +3,9 @@
     "license": "MIT",
     "type": "project",
     "description": "An app for storing data about servers",
+    "optimize-autoloader": true,
+    "classmap-authoritative": true,
+    "apcu-autoloader": true,
```

Comment:
The point is there is no need to add this here.

When installing on server we use `composer install --optimize-autoloader --apcu-autoloader --classmap-authoritative`